src/Schema/Plugin/SortPlugin.php

```diff
@@ -86,17 +87,67 @@ public function apply(Field $field, Schema $schema, array $config = []): void
     public static function sort(array $context): Closure
     {
         $fieldName = $context['fieldName'];
-        return function (?Sortable $list, array $args) use ($fieldName) {
+        return function (?Sortable $list, array $args, array $context, ResolveInfo $info) use ($fieldName) {
             if ($list === null) {
                 return null;
             }
-            $sortArgs = $args[$fieldName] ?? [];
+            $sortArgs = static::getSortArgs($info, $args, $fieldName);
```

Comment:
```suggestion
            $sortArgs = self::getSortArgs($info, $args, $fieldName);
```
Since it's a private method, `static` is effectively the same as `self` in this scenario - this is a bit of a nitpick but it's nicer to explicitly call private statics on `self` to avoid the appearance that there's some magic allowing a different method to be called on the subclass.

Reply:
Updated